wrappers/winuser.go

```diff
@@ -239,21 +240,38 @@ func GetForegroundWindow() syscall.Handle {
 	return syscall.Handle(r1)
 }
 
-func GetWindowText(hwnd syscall.Handle, buffer *uint16, maxCount int) (int, error) {
+func GetWindowText(hwnd syscall.Handle, buffer *uint16, maxCount int32) (int, error) {
 	r1, _, e1 := syscall.Syscall(
-		procGetWindowText.Addr(),
+		procGetWindowTextW.Addr(),
 		3,
-		uintptr(hwnd), uintptr(unsafe.Pointer(buffer)),
+		uintptr(hwnd),
+		uintptr(unsafe.Pointer(buffer)),
 		uintptr(maxCount))
 	if e1 != ERROR_SUCCESS {
 		return int(r1), e1
 	}
```

Comment:
We should check the return value to determine success or failure and only return `e1` if there's a failure:
```
if r1 == 0 {
    if e1 != ERROR_SUCCESS {
        return 0, e1
    } else {
        return 0, syscall.EINVAL
    }
}
```
Special handling will be needed for `GetWindowTextLength`, since it can return 0 to indicate either a failure or a text length of 0.